gpu/people/src/cuda/nvidia/NCV.hpp

```diff
@@ -430,7 +430,7 @@ class NCV_EXPORTS NCVMemStackAllocator : public INCVMemAllocator
 
     explicit NCVMemStackAllocator(Ncv32u alignment);
     NCVMemStackAllocator(NCVMemoryType memT, std::size_t capacity, Ncv32u alignment, void *reusePtr=nullptr);
-    ~NCVMemStackAllocator() override;
+    virtual ~NCVMemStackAllocator();
```

Comment:
From the [official docs](https://en.cppreference.com/w/cpp/language/override), `override` is a much better choice than `virtual`
> `override` ensures that the function is virtual **and** is overriding a virtual function from a base class
(emphasis mine)

Reply:
I agree, but these were throwing errors saying something like "marked `override` but does not override".

I'll double-check.

Reply:
OK, the `INCVMemAllocator` abstract base class has a pure virtual destructor, so there is no implementation to override, hence the compilation error when I declare the derived class destructors `override`.

Normally, you only make the destructor pure virtual if the ABC has no other pure virtual methods (which seems like a weird use case).

Reply:
Everything else in `INCVMemAllocator` is pure virtual, I can declare its destructor `virtual ~INCVMemAllocator() = default` and declare the derived class destructors `override`

Reply:
I didn't take a look at the code, but I have a handy godbolt: https://godbolt.org/z/8f99dG